apps/src/templates/instructions/TopInstructionsCSP.jsx

```diff
@@ -59,6 +53,14 @@ var styles = {
   embedView: {
     height: undefined,
     bottom: 0
+  },
+  paneHeaderOverride: {
+    color: color.default_text,
+  },
+  title: {
+    textAlign: 'center',
+    height: HEADER_HEIGHT + 'px',
```

Comment:
px should be unnecessary for height (but `lineHeight` is one of the few fields where it is required).